server/enterprise/message_export/message_export.go

```diff
@@ -111,10 +111,11 @@ func (m *MessageExportInterfaceImpl) RunExport(rctx request.CTX, exportType stri
 		return warningCount, model.NewAppError("RunExport", "ent.compliance.run_export.template_watcher.appError", nil, "", http.StatusAccepted).Wrap(err)
 	}
 	jobParams := shared.BackendParams{
-		Config:        m.Server.Config(),
-		Store:         m.Server.Store(),
-		FileBackend:   fileBackend,
-		HtmlTemplates: t,
+		Config:                m.Server.Config(),
+		Store:                 m.Server.Store(),
+		FileAttachmentBackend: fileBackend,
+		ExportBackend:         fileBackend,
```

Comment:
Should these be different?

Reply:
@nickmisasi Yah, doesn't matter -- is removed in https://github.com/mattermost/mattermost/pull/28805